datasets/kilt_tasks/dataset_infos.json

```diff
@@ -0,0 +1 @@
kilt.jsonl": {"num_bytes": 98951, "checksum": "43f82439f15f3141d115c06fb0dc0697a2a6313705dbdec6668eaefe9925a2c3"}, "http://dl.fbaipublicfiles.com/KILT/trex-train-kilt.jsonl": {"num_bytes": 1752330104, "checksum": "724a8a0fd1f99fe9b15cf6309d7ce63dcb7cd31f33aea717ff26e6d6b6fbdd97"}, "http://dl.fbaipublicfiles.com/KILT/trex-dev-kilt.jsonl": {"num_bytes": 3803558, "checksum": "7ee1a2cd351d928d01ede1763ae8d6ae442a00b07deef80e347852f20a029e5f"}, "http://dl.fbaipublicfiles.com/KILT/trex-test_without_answers-kilt.jsonl": {"num_bytes": 895854, "checksum": "0a2c6a31ebea567e7a8b31c44bf2fb59dab601c8da6de7f3a53419c7710bdc43"}, "http://dl.fbaipublicfiles.com/KILT/structured_zeroshot-train-kilt.jsonl": {"num_bytes": 71444475, "checksum": "f666359fa2b23e75d700bd1f25efd12d3bf981b1be70d8ad2d268d67e9bb3a5d"}, "http://dl.fbaipublicfiles.com/KILT/structured_zeroshot-dev-kilt.jsonl": {"num_bytes": 2266707, "checksum": "dd120ba5b62c6499f4e79bf5bf5a908eb96b583ba9bf8c76a4535ac21c5dac8d"}, "http://dl.fbaipublicfiles.com/KILT/structured_zeroshot-test_without_answers-kilt.jsonl": {"num_bytes": 1216038, "checksum": "3353137527c6fe9e48910e2c4d94d3ac01352c52e90a618beaaa807968fd39da"}, "http://dl.fbaipublicfiles.com/KILT/aidayago2-train-kilt.jsonl": {"num_bytes": 70139831, "checksum": "398231ec4ee9fd5616456d049875e51b1e0709f00ac2d74398dd4480ae5647b2"}, "http://dl.fbaipublicfiles.com/KILT/aidayago2-dev-kilt.jsonl": {"num_bytes": 21061554, "checksum": "3f5e23e1af46c8671c9870e13ced13fd517029da4963ad9ff4b834c4b42deb52"}, "http://dl.fbaipublicfiles.com/KILT/aidayago2-test_without_answers-kilt.jsonl": {"num_bytes": 14436143, "checksum": "2ba0836d3f8c70126022a83fd78da0bf7c910fa7813996dd2afee5e38de63648"}, "http://dl.fbaipublicfiles.com/KILT/wned-dev-kilt.jsonl": {"num_bytes": 12868348, "checksum": "e8e91d120abcb3433c952541aebd8dbf0c6abe3c378bdd64cd6d2186738cfbaf"}, "http://dl.fbaipublicfiles.com/KILT/wned-test_without_answers-kilt.jsonl": {"num_bytes": 13295124, "checksum": "f7b220d404d474f617b00c4a030a87024835b72aff131016716b84feca4d95ee"}, "http://dl.fbaipublicfiles.com/KILT/cweb-dev-kilt.jsonl": {"num_bytes": 90228527, "checksum": "c98b2301818b53df2025b09801be095152244dbbe7242efbfffd018929bfb4bf"}, "http://dl.fbaipublicfiles.com/KILT/cweb-test_without_answers-kilt.jsonl": {"num_bytes": 100216209, "checksum": "18b078340bde25e42c7136c5bc80c2e3557803e40137b40230af5834e74b6c0d"}, "http://dl.fbaipublicfiles.com/KILT/wow-train-kilt.jsonl": {"num_bytes": 71861702, "checksum": "91e04200a31daadbc9178382b724dac9be88ea9e76cd7a668e7a9430fe67713a"}, "http://dl.fbaipublicfiles.com/KILT/wow-dev-kilt.jsonl": {"num_bytes": 2418241, "checksum": "3202148ecaf41e77d44a2dec2453452c1ff8b3d646727e2c55ae6d9024e8e6b6"}, "http://dl.fbaipublicfiles.com/KILT/wow-test_without_answers-kilt.jsonl": {"num_bytes": 1292018, "checksum": "f3303ec5d7def4cf178552a8ed17bd01cb23bfeccc3abde147bebfa4bade6d8b"}}, "download_size": 3067457126, "post_processing_size": 0, "dataset_size": 2393111387, "size_in_bytes": 5460568513}}
```

Comment:
Two minor comments:
1) It would be nice to keep the same order for the datasets as in the paper (FC,EL,SF,NQ,D)
2) wrong GitHub link https://facebookresearch.github.io/KILT/ -> https://github.com/facebookresearch/KILT